index.d.ts

```diff
@@ -0,0 +1,58 @@
+export interface Options {
+  /**
+   * Speed `10` has 5% lower quality, but is about 8 times faster than the default. Speed `11` disables dithering and lowers compression level.
```

Comment:
This is missing the `1 (brute-force) to 11 (fastest)` part of the docs.

Reply:
I couldn't see in the style guide how you'd like this added? Just `Values: 1 (brute-force) to 11 (fastest)` on a line underneath or maybe in a `@remarks`?

Reply:
I think it's fine to just put it below here on a new line, without `@remarks`.